### PR TITLE
feat(stonedb): add cicd workflow of stonedb (#19)

### PR DESCRIPTION
**Pull request by CodingSuen**
## Summary about this PR

Issue Number: close #28 

## Tests Check List

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [ ] Improvement
- [ ] Performance Improvement
- [x] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

**Comment by CodingSuen** [comment]
@RingsC

File: .github/workflows/integration-compile-mtr.yml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+          sudo chmod -R 755 /stonedb56
+          sudo apt-get update
+          sudo apt-get install -y perl automake
+          cd /stonedb56/install
+          sudo /stonedb56/install/reinstall.sh
+
+      - name: check status
+        run: sudo netstat -lntp | grep mysql
+      
+      #- name: install valgrind
```
Comment (hustjieke) [inline .github/workflows/integration-compile-mtr.yml]: rm the commentted code

**Comment by hustjieke** [comment]
rm the pack and unpack artifact.

**Comment by hustjieke** [comment]
Please related to correct issue number, the ci/cd issue number is #19

File: .github/workflows/integration-compile-mtr.yml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          -DMYSQL_TCP_PORT=3306 \
+          -DENABLED_LOCAL_INFILE=1 \
+          -DEXTRA_CHARSETS=all \
+          -DDEFAULT_CHARSET=utf8 \
+          -DDEFAULT_COLLATION=utf8_general_ci \
+          -DWITH_BOOST=/usr/local/stonedb-boost/include
+          sudo make VERBOSE=1 -j`nproc`
+          sudo make install -j`nproc`
+        
+      - name: pack
```
Comment (hustjieke) [inline .github/workflows/integration-compile-mtr.yml]: I do not think we should use artifact.

File: .github/workflows/integration-compile-mtr.yml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        with:
+          name: stonedb-pack
+          path: ./
+          
+      - uses: geekyeggo/delete-artifact@v1
+        with:
+          name: stonedb-pack
+          failOnError: false
+      
+      - name: unpack
```
Comment (hustjieke) [inline .github/workflows/integration-compile-mtr.yml]: rm the pack and unpack steps

**Comment by hustjieke** [comment]
LGTM
PTAL @RingsC

**Comment by CodingSuen** [comment]
PTAL @RingsC

**Comment by RingsC** [comment]
LGTM， Merged.

**Pull request merged**